vscode-dotnet-runtime-library/src/EventStream/LoggingObserver.ts

```diff
@@ -6,31 +6,38 @@ import * as fs from 'fs';
 import { IEvent } from './IEvent';
 import { ILoggingObserver } from './ILoggingObserver';
 
-export class LoggingObserver implements ILoggingObserver {
+export class LoggingObserver implements ILoggingObserver
+{
     private log: string[] = [];
 
     constructor(private readonly logFilePath: string) {}
 
-    public post(event: IEvent): void {
-        this.writeLine(`${ new Date().toLocaleString() } ${ event.eventName }`);
+    public post(event: IEvent): void
+    {
+        this.writeLine(`${new Date().toLocaleString()} ${new Date().getMilliseconds()} ${event.eventName}`);

```

Comment:
Is this only including the MS count? Can you show a sample of what the entry looks like. Just from Wouldn't that just write

```
4/1/2025 0001
...
4/1/2025 0001 (this happened 5 seconds later and MS wrapped).
```

Also, does the output pad the MS count with leading 0s so you have fixed output width or does that not matter?

Reply:
```
4/1/2025, 1:42:28 PM 327 DotnetCommandSucceeded
uninstall
```
This is what it will look like in the current state. It's not the prettiest, though I did it in a rush since the performance investigation team added this as a last-minute request, since their setup makes it hard to test CDK performance with a test build of ours.

I think I will also add improving this to a list of things to do after this is merged. The output is not padded, though this is currently only read by humans, including during the perf investigation CDK is undergoing.